runtime/onert/backend/cpu/ops/ConvolutionLayer.cc

```diff
@@ -138,6 +154,26 @@ void ConvolutionLayer::configure(const Tensor *input, const Tensor *kernel, cons
 
 void ConvolutionLayer::run()
 {
+  if (_input->is_dynamic() || _kernel->is_dynamic())
+  {
+    const auto ifm_shape = getShape(_input).asFeature(_current_op_seq_layout);
```

Comment:
You have to use backend layout. But `_current_op_seq_layout` is frontend layout. You can use _input->layout() instead of `_current_op_seq_layout`

Reply:
Done.